src/views/transaction/components/fees-table/fees-table.view.jsx

```diff
@@ -47,14 +48,16 @@ function FeesTable({ l2Fee, estimatedWithdrawFee, token, preferredCurrency, fiat
       <div className={classes.feesTable}>
         <TransactionInfoRow
           title="Hermez fee"
-          subtitle={`${CurrencySymbol[preferredCurrency].symbol}${getL2FeeInFiat()}`}
+          subtitle={<FiatAmount amount={getL2FeeInFiat()} currency={preferredCurrency} />}
           hint="Step 1"
           value={`${Number(l2Fee.toFixed(MAX_TOKEN_DECIMALS))} ${token.symbol}`}
         />
         <TransactionInfoRow
           title="Ethereum fee (estimated)"
           hint="Step 2"
-          subtitle={`${CurrencySymbol[preferredCurrency].symbol}${getEstimatedWithdrawFeeInFiat()}`}
+          subtitle={
+            <FiatAmount amount={getEstimatedWithdrawFeeInFiat()} currency={preferredCurrency} />
```

Comment:
The function `getEstimatedWithdrawFeeInFiat` has the same problem described [here](#r728298357).